### PR TITLE
Fixes #7393 - searching users by role_id not supported

### PR DESCRIPTION
**Pull request by shlomizadok**
Search users by role_id

### CODE REVIEW

**Comment by domcleal** [comment]
Please remember to keep commit messages describing the change, not the old state... role_id searching _is_ supported.

**Comment by shlomizadok** [comment]
[test]

File: app/models/user.rb [hunk 1]
```diff
@@ -100,6 +100,7 @@ def self.name_format
   scoped_search :on => :admin, :complete_value => { :true => true, :false => false }, :ext_method => :search_by_admin
   scoped_search :on => :last_login_on, :complete_value => :true, :only_explicit => true
   scoped_search :in => :roles, :on => :name, :rename => :role, :complete_value => true
+  scoped_search :in => :roles, :on => :id, :rename => :role_id, :complete_value => true
```
Comment (ohadlevy) [inline app/models/user.rb]: I would prefer not to complete_value on ids, not just here, but across the app.

Comment (shlomizadok) [inline app/models/user.rb]: I'd move it from here. Will open a new issue to remove complete_value from across the app.

**Comment by isratrade** [comment]
Merged as 6e948c947f8e85dfe5a499d16b5704bddd34c58d. Thanks @shlomizadok !

**Pull request closed without merging**